Makefile

```diff
@@ -46,12 +46,15 @@ install-service: check-prev-proxies
 
 	install -D -m 755 trigger-pac4cli $(DESTDIR)/etc/NetworkManager/dispatcher.d/trigger-pac4cli
 	install -D -m 755 pac4cli.sh $(DESTDIR)/etc/profile.d/pac4cli-proxy.sh
+	install -D -m 644 pac4cli.config $(DESTDIR)/etc/pac4cli/pac4cli.config
 
 install-bin:
 	install -D -m 755 main.py $(DESTDIR)$(bindir)/pac4cli
 	@sed -i -e '1s+@PYTHON@+'$(PYTHON)'+' $(DESTDIR)$(bindir)/pac4cli
 
 	install -D -m 644 pac4cli.py $(DESTDIR)$(pythonsitedir)/pac4cli.py
+	install -D -m 644 wpad.py $(DESTDIR)$(pythonsitedir)/wpad.py
+	install -D -m 644 servicemanager.py $(DESTDIR)$(pythonsitedir)/servicemanager.py
```

Comment:
I hesitate a bit to put these modules at the top level includedir. They are very specific to pac4cli but the name doesn't show it. Maybe we should introduce a subdir named pac4cli?

Actually, my preference would be to just put it in a single file for now, but as you prefer. As you maybe know, I'm a firm believer in postponing modularization until the problem is fairly well understood. But maybe we understand it by now :)

I'll merge this so as not to block your progress.